tests/test_template.py

```
@@ -56,3 +55,31 @@ def test_template_renders_lib_flavour(tmp_path: Path, copier: CopierFixture) ->
     )
     assert (project / "src" / "lib.rs").exists()
     project.run("cargo build")
+
+
+def test_makefile_validates(tmp_path: Path, copier: CopierFixture) -> None:
+    """Generated Makefile validates with mbake."""
+    project = copier.copy(
+        tmp_path,
+        project_name="MakefileExample",
+        package_name="makefile_example",
+        license_year=datetime.now().year,
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)